.github/pull_request_template.md

```diff
@@ -1,21 +1,30 @@
 ## Pull Request Info
 
-### Jira
-
-- https://jira.mongodb.org/browse/DOCSP-NNNNN
-
-### Staged Changes
+Jira ticket: https://jira.mongodb.org/browse/DOCSP-NNNNN
+Staged changes:
 
 - [PAGE_NAME](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/BRANCH_NAME/)
 
+<!--
+- Write a summary of the changes and the related issue
+- Include relevant motivation and context
+- List any required dependencies for this change
+-->
+
 ### Reminder Checklist
 
-You might need to also update some corresponding pages. Check if completed or N/A.
+Before merging your PR, make sure to check a few things.
+
+- [ ] Create a Jira ticket for corresponding docs-realm update(s), if any
+- [ ] In your final commit message, summarize your PR for the docs release notes
+
+Final commit message template:
 
-- [ ] Create Jira ticket for corresponding docs-realm update(s), if any
-- [ ] Checked/updated Admin API
-- [ ] Checked/updated CLI reference
+```
+- Define Data Access Permissions
+  - Data Access Role Examples: Update CRUD Permissions example screenshots and copyable JSON
+```
 
 ### Review Guidelines
 
-[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)
+[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)
```

Comment:
Is there a reason we're changing this to the docs-realm Reviewing.md instead of referring to the one here in the docs-app-services repo?

Reply:
Oops! Copy-pasta mistake. Thanks for catching this.